src/Sleep.js

```diff
@@ -1,38 +1,25 @@
-class Sleep {
+import StatsParent from '../src/StatsParent'
+class Sleep extends StatsParent{
   constructor(sleepData, userID) {
+    super();
     this.sleepData = sleepData;
     this.userID = userID;
-  }
-
-  findUser() {
-    return this.sleepData.filter(user => user.userID === this.userID);
-  }
-
-  returnWeekOfData(week, userData) {
-    return [...userData].splice((-7 * week), 7);
-  }
-
-  returnWeek(week) {
-    var specificUser = this.findUser()
-    return [...specificUser].splice(-7 * week, 7).map(day => day.date);
+    this.finder = this.findUser(this.sleepData,this.userID)
   }
 
   returnAvgSleepData(relevantProperty) {
```

Comment:
This function can be taken out, it was replaced with returnAvgData(userData, id, relevantProperty) in the StatsParent.  In the DOM, it will just need to be called with sleepData as the argument for the userData param.